token-lending/program/src/processor.rs

```diff
@@ -2034,6 +1991,18 @@ fn process_update_reserve_config(
         );
         return Err(LendingError::InvalidMarketAuthority.into());
     }
+
```

Comment:
This is a bit hacky, and I don't love it.
The logic is basically
```
If 1 optional address is supplied, assume it's [switchboard]
If 2 optional addresses are supplied, assume its [pyth_product, pyth price]
If 3 optional addresses are supplied, assume it's [pyth_product, pyth_price, switchboard]
```

Reply:
i think perhaps it would be better if always had to pass 3 things but we could check if they were like default values

Reply:
Yeah thats a better idea.

Reply:
Did something slightly different.
I made it such that if new addresses aren't specified, then it calls the instruction with the old oracle addresses and a bogus address as the pyth product address.